### ISSUE TITLE
Refresh token from url

### ISSUE THREAD

**Issue by Doser86** [djtimca/hagooglewifi#91]
This is what I get when trying to get refresh token:

This website is not available

**Comment by Specter1975** [comment]
I'm having the same issue

**Comment by nathbullen** [comment]
Me as well. I just moved house and reset everything, now can't re-add this integration - even dug out an old refresh token from a previous backup, but I'm assuming as it's a new network config that token isn't valid anymore - it just spins forever and never completes the setup

**Comment by Nattle** [comment]
there's a chrome addon, but that appears to have stopped working in the past few days.

**Comment by Tankyspanky** [comment]
Same problem here - site is dead

**Comment by kedarv** [comment]
yup looks like the site isn't working. any other way to get this token?

**Comment by ensonic** [comment]
This [tool](https://github.com/joep1000/nest-rebooter) can get the tooken too.
